docs: update FlowFuse Expert docs for 2.31 release (#7378)
FlowFuse/flowfuse: docs/user/expert/chat.md

The more specific your prompt, the closer the result will be to what you need. Include node names, topic paths, endpoints, or field names where you have them. Expert will use them directly rather than substituting placeholders.

> > **Availability:** This feature is in open beta on **FlowFuse Cloud** (Starter, Team, and Enterprise tiers) — no request needed. It is now also available on **Self-Hosted Enterprise**; [contact us](https://flowfuse.com/contact-us/?subject=FlowFuse%20Expert%20Application%20Building) to get it set up. Agentic flow building works on both Hosted and Remote Instances.

Reviewer: I'll need somebody from the Crows team to validate this

@Steve-Mcl  or @dimitrieh

Reviewer: I don't think this is true

> and new self-hosted customers get it enabled automatically

They still need to request tokens for assistant and configure the expert MQTT bridge

Reviewer 2: > They still need to request tokens for assistant and configure the expert MQTT bridge

This is correct for SH

Reviewer 3: resolved with https://github.com/FlowFuse/flowfuse/pull/7378/commits/67bc17f7822adaec0e86cda2e3c46636ce8adaa5

> > Availability: This feature is in open beta on FlowFuse Cloud (Starter, Team, and Enterprise tiers), no request needed. It is also available on Self-Hosted Enterprise: [contact us](https://flowfuse.com/contact-us/?subject=FlowFuse%20Expert%20Application%20Building) to get it set up. Self-hosted setup is not automatic; it requires provisioning assistant tokens and configuring the Expert MQTT bridge. Agentic flow building works on both Hosted and Remote Instances, and requires the nr-assistant plugin at v0.16.0 or newer. FlowFuse Expert will let you know when an update is available.

Reviewer 3: ```suggestion
> Availability: This feature is in open beta on FlowFuse Cloud (Starter, Team, and Enterprise tiers), no request needed. It is also available on Self-Hosted Enterprise: [contact us](https://flowfuse.com/contact-us/?subject=FlowFuse%20Expert%20Application%20Building) to get it set up. Self-hosted setup is not automatic; it requires a provisioning token and configuring the bridge. Agentic flow building works on both Hosted and Remote Instances, and requires the nr-assistant plugin at v0.16.0 or newer. FlowFuse Expert will let you know when an update is available in your instance's immersive editor.
```

> > Availability: This feature is in open beta on FlowFuse Cloud (Starter, Team, and Enterprise tiers), no request needed. It is also available on Self-Hosted Enterprise: [contact us](https://flowfuse.com/contact-us/?subject=FlowFuse%20Expert%20Application%20Building) to get it set up. Self-hosted setup is not automatic; it requires a provisioning token and configuring the bridge. Agentic flow building works on both Hosted and Remote Instances, and requires the nr-assistant plugin at v0.16.0 or newer. FlowFuse Expert will let you know when an update is available in your instance's immersive editor.

Reviewer: ```suggestion
> Availability: This feature is in open beta on FlowFuse Cloud (Starter, Team, and Enterprise tiers), no request needed. It is also available on Self-Hosted Enterprise: [contact us](https://flowfuse.com/contact-us/?subject=FlowFuse%20Expert%20Application%20Building) to get it set up. Self-hosted setup requires a provisioning token and configuring the bridge,  use the contact us link to request a token. Agentic flow building works on both Hosted and Remote Instances, and requires the nr-assistant plugin at v0.16.0 or newer. FlowFuse Expert will let you know when an update is available in your instance's immersive editor.
```

Removed the "is not automtic; it ", it to me just reads more fluidly and pointed out how to request the token and configuration details.

Reviewer 3: ```suggestion
> Availability: This feature is in open beta on FlowFuse Cloud (Starter, Team, and Enterprise tiers), no request needed. It is also available on Self-Hosted Enterprise: [contact us](https://flowfuse.com/contact-us/?subject=FlowFuse%20Expert%20Application%20Building) to get it set up as Self-hosted requires a provisioning token and the bridge to be configured on the instance. Agentic flow building works on both Hosted and Remote Instances, and requires the nr-assistant plugin at v0.16.0 or newer. FlowFuse Expert will let you know when an update is available in your instance's immersive editor.
```
